openqa-label-known-issues

```diff
@@ -136,15 +150,9 @@ investigate_issue() {
     echo "$reason" >> "$out"
     if [[ "$curl_output" != "200" ]] && [[ "$curl_output" != "301" ]]; then
         # if we can not even access the page it is something more critical
-        handle_unreachable "$testurl" "$out" || return
+        handle_unreachable "$testurl" "$out" || return 0
```

Comment:
I noticed during my debugging that we have to explicitly return 0 here. Otherwise it would return the 1 from inside `handle_unreachable`.

Reply:
wonder if there should be a test for this

Reply:
If you remove the 0, the tests will fail